src/components/cms/BrowseSkill/skillLoader.js

```diff
@@ -7,6 +7,8 @@ const Container = styled.div`
   display: flex;
 `;
 
+const isMobile = isMobileView();
```

Comment:
sir, I have added it here
so that this variable is available to both functions instead of just one function

Reply:
@anishagg17 Adding it here, won't update the values on re-render.